### PR TITLE
Changed more versions to use the prerelease branch.

### PR DESCRIPTION
**Pull request by ruiterr**
This changes more versions in the partial-checkout sample to use the pre-release versions to make sure a local npm install works.

@DLehenbauer

### CODE REVIEW

**Review by DLehenbauer — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,18248 @@
 {
   "name": "root",
   "version": "0.14.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (znewton) [inline package-lock.json:4]: Have we officially moved to NPM 7 now?

Comment (DLehenbauer) [reply package-lock.json:4]: No, that wasn't an intentional decision.  I'll regenerate the lockfile and post a PR in a moment.  Thanks for letting me know.

Comment (DLehenbauer) [reply package-lock.json:4]: PR #6473

Comment (znewton) [reply package-lock.json:4]: Thank you for the quick reply and fix!

**Pull request merged**